### PR TITLE
fix dark theme blacking out text

### LINKED ISSUE #571: Dark Theme bug
**Describe the bug**

when using Roslynpad in dark mode (system OS),  controls, i.e textboxes are blacked out

**To Reproduce**
1. download Roslynpad or build from source 
2. in the code editior create a new file and attempt save the file, or use sysmbol Renaming feature 

https://github.com/user-attachments/assets/1e925353-4955-470d-85ca-0621191761ab

https://github.com/user-attachments/assets/c750892f-d356-4d89-931d-028971e680bd

https://github.com/user-attachments/assets/eeca8cc5-3c2b-4e84-ae4e-0932a8e2d0cb

https://github.com/user-attachments/assets/09419075-a303-4efe-9df4-755f0c4583c9

**Expected behavior**

**Version**
- OS: Windows 11 
- Build (or commit ID if built from source): Roslynpad20

I can make a pull request with the fixes

### PR DESCRIPTION
**Pull request by Randle-Lanre**
fixes #571

### CODE REVIEW

**Review by aelij — COMMENTED** [review]
Thanks for the fix and the detailed bug! Left a few comments

File: src/RoslynPad.Themes/Themes/dark_modern.json [hunk 1]
```diff
@@ -110,6 +110,7 @@
 		"terminal.tab.activeBorder": "#0078D4",
 		"textBlockQuote.background": "#2B2B2B",
 		"textBlockQuote.border": "#616161",
+        "textBox.background": "#FFFFFF",
```
[the remark this answers is not in the document]
Comment (Randle-Lanre) [reply src/RoslynPad.Themes/Themes/dark_modern.json]: thanks for the pointing me to https://themes.vscode.one/, I have made the requested changes.

File: src/RoslynPad/ThemeDictionary.cs [hunk 2]
```diff
@@ -51,7 +51,7 @@ public ThemeDictionary(Theme theme) : base(theme)
     public const string TabBorder = "tab.border";
     public const string InputBorder = "input.border";
     public const string TitleBarActiveBackground = "titleBar.activeBackground";
-
+    public const string TextBoxBackground = "textBox.background";
```
[the remark this answers is not in the document]
Comment (Randle-Lanre) [reply src/RoslynPad/ThemeDictionary.cs]: done

File: src/RoslynPad/DocumentTreeView.xaml [hunk 3]
```diff
@@ -148,6 +148,7 @@
                              BorderThickness="0"
                              Validation.ErrorTemplate="{StaticResource ErrorTemplate}"
                              Text="{Binding SearchText, UpdateSourceTrigger=PropertyChanged, ValidatesOnNotifyDataErrors=True}"
+                             Foreground="White"
```
[the remark this answers is not in the document]
Comment (Randle-Lanre) [reply src/RoslynPad/DocumentTreeView.xaml]: requested changes have been applied

**Comment by aelij** [comment]
Thanks for the fix!

**Pull request merged**